routes/api.php

```diff
@@ -17,3 +18,5 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::get('/game/{id}', [AccountController::class, 'toggleWishlist']);
```

Comment:
any particular reason you chose this route path? it's very similar to the game-details route path and could be confusing

Reply:
I actually don't recall how I came to this path actually! I will look in to this